superbench/benchmarks/micro_benchmarks/directx_gemm_flops_performance/GPUCore.cpp

```diff
@@ -0,0 +1,507 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// GPUCore.cpp : This file contains the 'main' function. Program execution begins and ends there.
+#include <algorithm>
+#include <array>
+#include <iostream>
+#include <tchar.h>
+#include <vector>
+
+#include <directml.h>
+
+#include "GPUCore.h"
+
+#define DEBUG 0
```

Comment:
how about move the PreprocessorDefinitions into vcxproj section so if you build DEBUG version, the debug will set, and if you build RELEASE version, the debug will not set

Reply:
good idea, moved to PreprocessorDefinitions in vcxproj